<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interviewer_operations</class>
 <widget class="QDialog" name="interviewer_operations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="interviewee_info_table">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>717</width>
     <height>206</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft PhagsPa</family>
    </font>
   </property>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>5</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>姓名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>学号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>专业班级</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>第一志愿</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>第二志愿</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>是否服从调剂</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="information">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正剪纸繁体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>456</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正剪纸繁体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>面试官打分</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>490</y>
     <width>601</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>方正剪纸繁体</family>
       </font>
      </property>
      <property name="text">
       <string>编号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="score_id_line"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>方正剪纸繁体</family>
       </font>
      </property>
      <property name="text">
       <string>分数</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="score_line"/>
    </item>
    <item>
     <widget class="QPushButton" name="confirm_score">
      <property name="font">
       <font>
        <family>方正剪纸繁体</family>
       </font>
      </property>
      <property name="text">
       <string>确认打分</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="search_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>查询面试信息</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
